usersync/chooser.go

```diff
@@ -136,7 +140,12 @@ func (c standardChooser) Choose(request Request, cookie *Cookie) Result {
 }
 
 func (c standardChooser) evaluate(bidder string, syncersSeen map[string]struct{}, syncTypeFilter SyncTypeFilter, privacy Privacy, cookie *Cookie) (Syncer, BidderEvaluation) {
-	syncer, exists := c.bidderSyncerLookup[bidder]
+	coreBidder, exists := c.normalizedBidderNamesLookup(bidder)
+	if !exists {
+		return nil, BidderEvaluation{Status: StatusUnknownBidder, Bidder: bidder}
+	}
+
+	syncer, exists := c.bidderSyncerLookup[coreBidder.String()]
 	if !exists {
 		return nil, BidderEvaluation{Status: StatusUnknownBidder, Bidder: bidder}
```

Comment:
Line 145 and 150 returns same error.  How about changing Line 150 to `syncerKeyNotFound` status

Reply:
`syncerKeyNotFound ` seems irrelevant to me here since this check is not related with syncher keys. 
This is actually checking if there is a coreBidder exists for the provided bidder in the request to which we are returning `UnknownBidder` if there is no coreBidder found for the provided bidder and that sounds ok to me.